projects/cmake/build.sh

```diff
@@ -153,12 +152,12 @@ fi
     fi
 
     echo "Running './regenerate.sh $(pwd)/$BUILD_DIR"
-    ./regenerate.sh $(pwd)/$BUILD_DIR
+    bash regenerate.sh $(pwd)/$BUILD_DIR
```

Comment:
What's the rationale here?

Should we change regenerate.sh to start with `#!/bin/bash`?  What about it is bash-specific?

Reply:
Currently regenerate has `#!/bin/sh`, which doesn't include some features that bash has...

So, maybe remove this change and modify regenerate.sh, and then if we get a problem with that, fix it then.

Reply:
OK, for now I've reverted this change and we see when it pops up again ...